### DISCUSSION TITLE
[discussion] swow 能与 workerman 搭配使用吗？

### DISCUSSION THREAD

**Discussion by lanlin** [walkor/Workerman#747] [Q&A]
想将 workerman 与 swow 结合起来使用，但是貌似有些问题，水平有限，搞不懂其中的原因，希望大佬帮忙看看

相同的讨论也在 swow 项目中提出了，链接如下：

https://github.com/swow/swow/issues/115

**Comment by walkor** [comment]
https://github.com/walkor/workerman/blob/master/src/Events/Swoole.php  类似这样给workerman加一个events适配就好了。
现在看 swow 好像没有完整的文档，不清楚是否像swoole那样支持Event接口。

**Comment by lanlin** [comment]
@walkor 好的，我试试看，目前确实是没文档，不清楚是不是有这样的接口

**Comment by lanlin** [comment]
@walkor 找到原因了，swow 貌似和 event 扩展有冲突。我把扩展禁用就好了。
但是 stream_socket_client() 中如果加了 STREAM_CLIENT_PERSISTENT 也会出现连接不上的情况
